samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -1849,6 +1849,22 @@
           }
         ]
       }
+    },
+    "EventBridgePutEventsPolicy": {
+      "Description": "Gives permissions to send events to EventBridge",
+      "Parameters": {},
+      "Definition": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "events:PutEvents",
+              "events:PutPartnerEvents"
```

Comment:
I don't think it makes sense to include `PutPartnerEvents` in this policy template. The point of the policy template is to provide a shorthand syntax for common policies while allowing you to scope them down to a specific resource. So when we add EventBus as a parameter to this template, `PutPartnerEvents` no longer makes sense since that API doesn't allow you to specify a specific EventBus.

You could create a separate `EventBridgePutPartnerEventsPolicy` policy template if you think it's an important one to have.

Reply:
At the moment I will only limit the policy to put events, if there will be future need I think the `EventBridgePutPartnerEventsPolicy` can be added separately.